content/200-concepts/100-components/02-prisma-client/260-client-extensions/100-client-extensions-model.mdx

```diff
@@ -0,0 +1,135 @@
+---
+title: 'Add custom methods to your models (Prisma Client extensions)'
```

Comment:
"(Prisma Client extensions)" is repeated from the breadcrumb navigation visually. I think we could just leave this out.

Reply:
This is true, but I'm sceptical that everyone looks at the breadcrumbs. However, with fresh eyes on this, I agree that it should come out anyway. I think there's enough context in the page itself, our internal search makes the breadcrumbs very prominent. For external search, a person is either going to be searching for how to augment models (and initially they won't care how - they just want to know how to do it) or for client extensions, in which case they'll hopefully get the main extensions page as the top result. 

TL;DR - I'm removing this for slightly different reasons than the one you gave!